src/sonic-yang-mgmt/yang-models/sonic-port.yang

```diff
@@ -0,0 +1,90 @@
+module sonic-port{
+
+	namespace "http://github.com/Azure/sonic-port";
+	prefix port;
+
+	import ietf-yang-types {
+		prefix yang;
+	}
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	import sonic-head {
+		prefix head;
+		revision-date 2019-07-01;
+	}
+
+	organization "Linkedin Corporation";
+
+	contact "lnos_coders@linkedin.com";
+
+	description "PORT yang Module for SONiC OS";
+
+	revision 2019-07-01 {
+		description "First Revision";
+	}
+
+	container sonic-port{
+		container PORT {
+
+			description "PORT part of config_db.json";
+
+			list PORT_LIST {
+
+				key "port_name";
```

Comment:
Again two different versions. The committed version calls it ifname, here it is called port_name.

We need to sync.

Reply:
port should have port_name instead of ifname, we have interface TAble separately.